src/metabase/driver/postgres.clj

```diff
@@ -758,3 +760,29 @@
   [driver prepared-statement i t]
   (let [local-time (t/local-time (t/with-offset-same-instant t (t/zone-offset 0)))]
     (sql-jdbc.execute/set-parameter driver prepared-statement i local-time)))
+
+(defn- load-from-csv-sql [schema-name table-name column-names file-name]
+  (str "COPY " schema-name "." table-name "(" (str/join "," column-names) ") FROM '" file-name
+       "' WITH (FORMAT CSV, HEADER TRUE, ENCODING 'UTF8', QUOTE '\"', ESCAPE '\\')"))
```

Comment:
I'm not super familiar with how SQL injection works with the drivers. We are calling `u/slugify` on the column names, but do we need another check to ensure malicious CSVs don't cause problems?

(For that matter, we currently have zero checks on file names)